tox.ini

```diff
@@ -8,7 +8,7 @@ extras =
   test
   cplex
 commands =
-  pytest {posargs}
+  pytest {posargs} --ignore=circuit_knitting/utils/metrics.py --ignore=circuit_knitting/cutting/cutqc
```

Comment:
these options should be moved to `addopts` in `pyproject.toml` so they are active in the `coverage` environment as well.

Reply:
Done here: [dc31577](https://github.com/Qiskit-Extensions/circuit-knitting-toolbox/pull/527/commits/dc315775d8b77de35480026dd9fa0c62f124eeeb)

Reply:
Still need to remove the options here and revert my earlier change.

Reply:
discussion continued at https://github.com/Qiskit-Extensions/circuit-knitting-toolbox/pull/527#discussion_r1552126580